docker/build-container.sh

```
@@ -1,164 +1,164 @@
-        --build-arg BASE=${CONTAINER_TAG} \
-        --build-arg SD_IMAGE=${SD_IMAGE} --build-arg SD_TAG=${SD_TAG} \
-        --build-arg UID=${USER_UID} --build-arg GID=${USER_GID} \
-        -t ${CONTAINER_TAG} -t ${CONTAINER_LATEST} . ;;
-  esac
-
-  if [ "$PUSH_IMAGES" == "true" ]; then
-    docker push ${CONTAINER_TAG}
-    docker push ${CONTAINER_LATEST}
-  fi
-done
+#!/bin/bash
```

Shellcheck (0.11.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)